docs/Users_Guide/existing_builds.rst

```diff
@@ -0,0 +1,561 @@
+
+Existing Builds
+===============
+
+The METplus team supports the installation of the **METplus software components** 
+on several operational research high performance computing platforms. This 
+includes installations at NCAR, NOAA, and other select community machines.
+Pre-built METplus images on **DockerHub** are also provided.
+
+The **NCAR/RAL Common Installation** location under the **NCAR Machines**
+drop down menu is only updated after official releases.  No developmental releases 
+are installed in a common space on RAL machines.
+
+Please submit a new discussion in the 
+`METplus Components Discussion <https://github.com/dtcenter/METplus/discussions>`_
+forum if no instructions exist for the current release on a supported 
+platform and the release is needed on that platform.
+
+Select from the list below for instructions on using existing builds of 
+the METplus components' software packages. Please note that the commands to 
+load the METplus components assume the user is using bash. If an installation 
+is needed on a machine not listed here, please follow the 
+`installation instructions <https://metplus.readthedocs.io/projects/met/en/latest/Users_Guide/installation.html#software-installation-getting-started>`_ 
+in the 
+`MET User’s Guide <https://metplus.readthedocs.io/projects/met/en/latest/Users_Guide/>`_.
```

Comment:
```suggestion
`MET User’s Guide <https://met.readthedocs.io/en/latest>`_.
```
The current link you have is correct.  It is a new format where MET is listed as a sub project of METplus.  We haven't decided whether or not we want to use that or not, so I'd like to revert to the old URL, which still works, at this time.